ci/testgrid/config.yaml

```diff
@@ -50,27 +54,20 @@ default_dashboard_tab:
   code_search_url_template:      # The URL template to visit when searching for changelists
     url: https://github.com/knative/serving/compare/<start-custom-0>...<end-custom-0>
   alert_options:
-    alert_mail_to_addresses: 'knative-productivity-dev@googlegroups.com'
-
-#################################################################
-# Test groups
-#################################################################
-
+  alert_mail_to_addresses: 'knative-productivity-dev@googlegroups.com'
+######################################
```

Comment:
I think we should either have the full comment, or no comment. A single separator won't help with readability in the generated file.

Reply:
Removed the comment.